Vishal/137 updated pr template (#140)
mrjones2014/yaclt: changelogs/2021-09-10T20-11-51.999+05-30.md

> [NEW] A user-friendly description of your change {#137}

Reviewer: This should be `[IMPROVED]` and provide a description of your change -- this is the default placeholder changelog.

Author: ohh okaay

Author: Done  !! @mrjones2014

> [NEW] A user-friendly description of your change {#137}
> Upadated PR template to make sure people are reading the contributing guidelines. Made changes in the pull_request_template.md file.

Reviewer: I meant like this:

```suggestion
[IMPROVED] Upadated PR template to make sure people are reading the contributing guidelines. {#137}
```

Please read through the `README.md`, `COMMANDS.md`, and `CONTRIBUTING.md` to get an understanding of how the CLI works.

Reviewer: I've added it for you this time.
